### PR TITLE
Frontend initialized

### PR DESCRIPTION
**Pull request by jatin142002**
The initial setup of the frontend is done.

### CODE REVIEW

File: frontend/views/home.ejs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
+    <script src="https://code.jquery.com/jquery-2.1.0.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  </head>
+
+  <body>
+    <div id="topicsDict"><%=JSON.stringify(topicsDict)%></div>
```
Comment (Aksh-Bansal-dev) [inline frontend/views/home.ejs]: remove ejs code

File: frontend/js/home.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+      } else {
+        var emptyWarning = document.getElementById("emptyWarning");
+        emptyWarning.style.display = "none";
+        var preferred = document.getElementById("preferred");
+        preferred.style.display = "block";
+      }
+      var obj = { contests: contests, userId: userId };
+      fetch(`/fetchData/${JSON.stringify(obj)}`, {
+        method: "GET",
+      })
```
Comment (Aksh-Bansal-dev) [inline frontend/js/home.js]: Refactor code to remove duplication

File: frontend/js/home.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    //Hiding Extra information
+    var emptyWarning = document.getElementById("emptyWarning");
+    emptyWarning.style.display = "none";
+
+    var obj = { contests: contests, userId: userId };
+    fetch(`/fetchData/${JSON.stringify(obj)}`, {
+      method: "GET",
+    })
+      .then((response) => response.json())
+      .then((json) => {
```
Comment (Aksh-Bansal-dev) [inline frontend/js/home.js]: Make a global dummy url for lambda and take the schema as told earlier.

Comment (Aksh-Bansal-dev) [reply frontend/js/home.js]: POST `<lambda_endpoint>`
request:
```
body: {
  authorId: string
}
```
response:

```

{
  "<tag_name>": []Problem 
}
```
```
Problem {
  name string, 
  rating int,
  problem_url string,
}
```

**Review by Aksh-Bansal-dev — CHANGES_REQUESTED** [review]
We will not use `ejs`, instead we will use `html`.

Also, test this in local.

**Review by HimanShu0923 — APPROVED** [review]

**Pull request merged**